<!DOCTYPE html>
<html>
<head>
    <title>MutPy mutation report - mutation #2</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/bootstrap/3.0.3/css/bootstrap.min.css">
    
<link href="http://alexgorbatchev.com/pub/sh/current/styles/shCore.css" rel="stylesheet" type="text/css" />
<link href="http://alexgorbatchev.com/pub/sh/current/styles/shThemeDefault.css" rel="stylesheet" type="text/css" />

    <script src="https://code.jquery.com/jquery.js"></script>
    <script src="https://netdna.bootstrapcdn.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
    <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
    <script src="https://oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js"></script>
    <![endif]-->
    
<script src="http://alexgorbatchev.com/pub/sh/current/scripts/shCore.js" type="text/javascript"></script>
<script src="http://alexgorbatchev.com/pub/sh/current/scripts/shBrushPython.js" type="text/javascript"></script>
<script type="text/javascript">
    SyntaxHighlighter.all();
    window.setTimeout(function () {
        
        $('.line.number2').attr('title', 'AOR');
        
    }, 0);
</script>

</head>
<body>
    <div class="container">
        
<div class="page-header">
    <h1>Mutation #2</h1>
</div>
<h3>Details</h3>
<ul>
    <li>module - <code><module 'datasets.GPT_4.Chain_of_thought.programs.program_064' from 'C:\\Users\\skaspers\\OneDrive - Capgemini\\Documents\\Prompted Testsuite Analysis\\.\\datasets\\GPT_4\\Chain_of_thought\\programs\\program_064.py'></code></li>
    <li><span class="label label-success">killed</span> by <code>test_floating_point_numbers (datasets.GPT_4.Chain_of_thought.tests.test_064.TestVersion)</code></li>
    
    <li>duration - 0.001 s</li>
    
    
    <li>tests run - 2</li>
    
</ul>

<h3>Exception traceback</h3>
<pre>Traceback (most recent call last):
  File "C:\Users\skaspers\OneDrive - Capgemini\Documents\Prompted Testsuite Analysis\.\datasets\GPT_4\Chain_of_thought\tests\test_064.py", line 31, in test_floating_point_numbers
    self.assertEqual(max_product_tuple([(1.5, 2.5), (3.5, 4.5), (5.5, 6.5)]), 35.75)
AssertionError: 0.0 != 35.75
</pre>

<h3>Mutations</h3>
<ul>
    
    <li>AOR - line 2</li>
    
</ul>
<h3>Mutant</h3>
<pre class="brush: python; first-line: 1; highlight: [2]; toolbar: false;">def max_product_tuple(list1):
    result_max = max([abs(x // y) for (x, y) in list1])
    return result_max</pre>

    </div>
</body>
</html>